packages/patternfly-3/patternfly-react/src/components/DateTimePicker/TimePicker.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl, InputGroup, Icon, OverlayTrigger, Popover } from '../../';
+import TimeInput from './TimeComponents/TimeInput';
+
+class TimePicker extends React.Component {
+  getDateFromTime = time => {
+    if (Date.parse(time)) {
+      return new Date(time);
+    }
+    return new Date(`1/1/1 ${time}`);
+  };
+  state = {
+    value: this.getDateFromTime(this.props.value)
+  };
+  formatDate = () => {
```

Comment:
can date format support an optional prop ? (i.e 'yyyy-mm-dd HH:MM:SS`), in some cases the user needs to bypass localization.